libs/plugins/stream-client/src/lib/core/stream.service.ts

```diff
@@ -137,4 +144,42 @@ export class StreamService {
         map(() => itemId)
       );
   }
+
+  deleteStream(stream) {
+    return this.translate
+      .get(['STREAMS.DELETE-STREAM:CONFIRM_DELETE', 'MODAL:CONFIRM-DELETION'], {
+        streamName: stream.name,
+      })
+      .pipe(
+        switchMap(translation => {
+          return this.confirmationModal.openModal({
+            title: translation['MODAL:CONFIRM-DELETION'],
+            textMessage: translation['STREAMS.DELETE-STREAM:CONFIRM_DELETE'],
+          }).result;
+        }),
+        filter(result => result === ConfirmationResult.Confirm),
+        switchMap(() => {
+          return this.appResourceService.deleteResourceWithTrigger(stream.id, null).pipe(
```

Comment:
Why are we always calling the `deleteResourceWithTrigger` with `null` as second parameter?